### ISSUE TITLE
add buildkite

### ISSUE THREAD

**Issue by CarloLucibello** [CarloLucibello/GraphNeuralNetworks.jl#3]
for gpu CI

**Comment by CarloLucibello** [comment]
Per buildkite's admin answer to my inquiries, this is not doable as long as GNN.jl is privately owned and not part of an org. Therefore, it has to be done after #80. 
The instructions on how to setup buildkite are in
https://github.com/JuliaGPU/buildkite

**Comment by CarloLucibello** [comment]
@maleadt can we enable buildkite now that the package has been moved to an org?

**Comment by maleadt** [comment]
Yes, but you'll have to make me an admin of the org, temporarily. Best coordinate this on Slack.
